src/components/ControlInput.vue

```diff
@@ -465,7 +465,10 @@ export default {
         },
         submitForm() {
             let rawInput = this.$refs.input.getValue();
-            if (!rawInput) {
+
+            // dont send just whitespace messages clear the input instead
+            if (!rawInput.trim()) {
```

Comment:
I'm not sure we should be doing this. Empty lines are commonly used during conversation/jokes/ascii/etc and that's not something we should be preventing.